angr/state_plugins/loop_data.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import copy as c
```

Comment:
I would avoid renaming a common package to a one-letter alias - especially since you are only using it once!